iree/test/e2e/xla_ops/BUILD

```diff
@@ -125,6 +125,8 @@ iree_check_single_backend_test_suite(
             "while.mlir",
         ],
         include = ["*.mlir"],
+        # Exclude f16 tests as it is not supported by vmla.
+        exclude = ["*_f16.mlir"],
```

Comment:
This kind of contradicts the big comment at the top of this file saying that everything should run on VMLA. It is still our reference backend. I'm not sure if we still generate op coverage tables from this information, but that was part of the reason for standardized naming for these files. When I needed to do something similar for integer datatypes in matmul, I created a subdirectory for "partial" support: https://github.com/google/iree/commits/main/iree/test/e2e/xla_ops/partial. Maybe we should do that here.

Reply:
Makes sense. I'm not sure if we'll want to support float16 in VMLA I'll discuss this further with Ben and the rest of the team. I moved those tests in partial for now.